src/os.rs

```diff
@@ -54,7 +56,9 @@ fn next_u64(mut fill_buf: &mut FnMut(&mut [u8])) -> u64 {
 
 #[cfg(all(unix, not(target_os = "ios"),
           not(target_os = "nacl"),
-          not(target_os = "freebsd")))]
+          not(target_os = "freebsd"),
+          not(target_os = "netbsd"),
```

Comment:
netbsd have not specialized implementation.

I see several possibilities:
  - removing the line : netbsd will go back to use `/dev/urandom` (as in libstd)
  - using freebsd implementation: it will rely on `sysctl(KERN_ARND)` (untested: `KERN_ARND` is defined in NetBSD, but I didn't check if rust libc provide it)

I think the safer thing for now would be removing the line.